draft-ietf-moq-transport.md

```diff
@@ -521,6 +521,31 @@ which tracks are delivered via Subscribe Parameters, but the actual content of
 the tracks does not depend on those parameters; this is in contrast to
 protocols like HTTP, where request headers can alter the server response.
 
+### Subscriptions
+
+A subscription can be initiated by either a publisher or a subscriber.  A
+publisher initiates a subscription to a track by sending the PUBLISH message.
+The subscriber either accepts or rejects the subscription using PUBLISH_OK or
+PUBLISH_ERROR.  A subscriber initiates a subscription to a track by sending the
+SUBSCRIBE message.  The publisher either accepts or rejects the subscription
+using SUBSCRIBE_OK or SUBSCRIBE_ERROR.  Once either of these sequences is
+complete, the subscription can be updated by the subscriber using
```

Comment:
s/Once either of these sequences is complete/Once the subscription is successful,/...